app/src/main/java/me/quadphase/qpdex/databaseAccess/ExternalDbOpenHelper.java

```diff
@@ -117,13 +119,5 @@ public synchronized void close() {
     @Override
     public void onCreate(SQLiteDatabase db) {}
     @Override
-    public void onUpgrade(SQLiteDatabase db, int oldVersion, int newVersion) {
-        if (newVersion > oldVersion) {
-            try {
-                copyDataBase();
-            } catch (Exception e) {
-                // do something
-            }
-        }
-    }
+    public void onUpgrade(SQLiteDatabase db, int oldVersion, int newVersion) {}
```

Comment:
Why did we change this? Is there a way to version the database so that we only copy when needed using this method?